example/src/main/java/org/secondbase/example/main/HelloJsonLogging.java

```diff
@@ -12,7 +14,8 @@ public static void main(final String[] args) throws SecondBaseException {
                 "--service", "test",
                 "--datacenter", "dc1"
         };
-        new SecondBase(realArgs, new Flags().loadOpts(helloJsonLogging));
+        final SecondBaseModule[] modules = new SecondBaseModule[] { new JsonLoggerModule() };
+        new SecondBase(realArgs, modules, new Flags().loadOpts(helloJsonLogging));
```

Comment:
Examples that doesn't revolve around the use of Flags shouldn't have flags in them. Can we remove the flags in this class and add the Flag-less SecondBase constructor again?

SecondBase(final SecondBaseModule[] modules)
SecondBase(final SecondBaseModule[] modules, final Flags flags)